lib/board.rb

```diff
@@ -0,0 +1,35 @@
+require_relative './token'
+
+class Board
+    include Token
+    attr_accessor :size, :positions, :max_positions
+    
+    def initialize(size:)
+        @size = size
+        @max_positions = size * size
+        @positions = (1..max_positions).map {|num| num = empty }
+    end
+
+    def update(position:, token:, all_positions: nil)
```

Comment:
all_positions was something I passed in for easy testing.

so instead of updating one position at a time, I can pass in a new array during tests.

Should I just make a new function just for tests?